cypress/support/cypress-commands/commands.js

```diff
@@ -675,16 +672,6 @@ Cypress.Commands.add('launchApp', (appType, appCallSign) => {
         },
       },
     };
-    const messageIntent = {
-      action: CONSTANTS.SEARCH,
-      data: data,
-      context: { source: CONSTANTS.DEVICE },
-    };
-
-    requestMap = {
-      method: CONSTANTS.DISCOVERY_LAUNCH,
-      params: { [CONSTANTS.APP_ID]: appId, [CONSTANTS.INTENT]: messageIntent },
-    };
```

Comment:
Add comment before the if condition. 
// If testType == lifecycle, modifying data to include _lifecycle_validation = true_ in the intent to be sent to the app

Reply:
Added comment